src/sentry/static/sentry/app/views/settings/components/settingsNavigationGroup.tsx

```diff
@@ -24,6 +28,22 @@ const SettingsNavigationGroup = (props: NavigationGroupProps) => {
           ...(project ? {projectId: project.slug} : {}),
         });
 
+        const handleClick = () => {
+          //only call the analytics event if the URL is changing
+          if (analyticsParams && to !== window.location.pathname) {
+            //remvove any undefines to stop warnings
+            analyticsParams = omitBy(
```

Comment:
`omitBy` doesn't return the type I want so I have to typecast. I could cast it as `any` to avoid the warnings as well and save the import at the top if people think that's better.